Permanent/ViewControllers/SideMenuViewController.swift

```diff
@@ -230,6 +233,13 @@ extension SideMenuViewController: UITableViewDataSource, UITableViewDelegate {
             newRootVC.archiveData = archive
             AppDelegate.shared.rootViewController.changeDrawerRoot(viewController: newRootVC)
             
+        case .publicGallery:
+            guard let archive = viewModel?.getCurrentArchive() else { return }
+            let newRootVC = UIViewController.create(withIdentifier: .publicGallery, from: .main) as! PublicGalleryViewController
+            newRootVC.archive = archive
+            newRootVC.viewModel?.account
```

Comment:
I don't think the line above does anything

Reply:
newRootVC.archive is not used anymore.